ptsched.c: In function '_gg_task_thread_waiter':
ptsched.c:200: error: label at end of compound statement

--- libgii-0.9.0/gg/ptsched.c.orig	2004-05-16 11:35:42.000000000 +0200
+++ libgii-0.9.0/gg/ptsched.c	2005-02-04 10:41:35.461353744 +0100
@@ -196,7 +196,7 @@
 			goto skip; /* push/pop must be cleanly nested.       */
 		}
 		pthread_cond_wait(&_gg_task_thread.tick, &_gg_task_thread.mtx);
-	skip:
+	skip:	;
 		pthread_cleanup_pop(1);
 		if (done) return myid;
 		if (!_gg_task_thread.running) return myid;
